binaries/org.eclipse.swt.win32.win32.x86_64/build.properties

```diff
@@ -35,8 +35,14 @@ source.. = \
 	../../bundles/org.eclipse.swt/Eclipse SWT Browser/win32,\
 	../../bundles/org.eclipse.swt/Eclipse SWT OpenGL/win32,\
 	../../bundles/org.eclipse.swt/Eclipse SWT OpenGL/common
+source.test = \
+	../../bundles/org.eclipse.swt/Eclipse SWT Tests/win32
 output.. = bin/
+output.test = bin_test/
 
 pom.model.property.os=win32
 pom.model.property.ws=win32
 pom.model.property.arch=x86_64
+
+additional.bundles = junit-jupiter-api,\
```

Comment:
For test folder use a JUNIT Classpath container instead of additional bundles (what server a completely different purpose)

Reply:
I wanted to ask you exactly what you think is the proper approach here, as additional.bundles were only my first attempt to get things running. I have added this as a means to make Tycho have the JUnit dependencies at compile-/test-time. I've already specified a JUnit classpath container in the `.classpath`, but that's not considered for the Tycho build, is it? At least Tycho complains about missing JUnit dependencies when compiling the test sources. Probably it would be better to add the JUnit dependency directly to the Maven build?

Reply:
> I've already specified a JUnit classpath container in the .classpath, but that's not considered for the Tycho build, is it? 

This is supported since [Tycho 2.4](https://github.com/eclipse-tycho/tycho/blob/main/RELEASE_NOTES.md#support-resolving-of-junit-classpath-container)

> At least Tycho complains about missing JUnit dependencies

Maybe a bug or misconfiguration, at least we have some test for this [here](https://github.com/eclipse-tycho/tycho/tree/main/tycho-its/projects/compiler.junitcontainer) that *should* prove it works.

> Probably it would be better to add the JUnit dependency directly to the Maven build?

While it maybe works, the purpose of Tycho is to not need special maven configuration so it might be a good opportunity to find out why it does not work and either improve Tycho or [the documentation](https://tycho.eclipseprojects.io/doc/latest/TestingBundles.html#maven-surefire-plugin) (sadly it misses the part about eclipse setup currently).

Reply:
Then something does obviously not work here as expected:
* Tycho does not consider the test source folder specified via the `.classpath`. If I understand correctly, it should automatically use those test source folder to compile test classes.
* When specifying the test source and output folder manually in the build.properties, Tycho complain about missing JUnit dependencies when compiling them.

I guess that both the specification of test source and output folder in the build.properites as well as the additional specification of JUnit dependencies relate to the same root cause, due to which the information from the `.classpath` file are not considered correctly. I will try to figure out what is special in the SWT setup compared to the Tycho integration tests for that behavior.

Reply:
it might be because classpath is a linked file, can you try to copy the file as a regular one into the project? I recently added support for linked files, but that for sure not respected everywhere at the moment.

Reply:
Thank you, @laeubi! The linked files are actually the problem here. It affects both the linked `.classpath` file as well as the linked test source folder.
* When replacing the linked `.classpath` with the same file directly placed in the project, Tycho properly considers the test source folder specified in the `.classpath` file. Then it only complains that it does not find that (linked) folder:
`[WARNING] Source directory C:\dev\eclipse\platform-only\git\eclipse.platform.swt\binaries\org.eclipse.swt.win32.win32.x86_64\Eclipse SWT Tests\win32 does not exist`
* When also replacing the linked source test folder with the same folder directly placed in the project, test source files are compiled correctly and tests are executed.

Reply:
okay then this needs some adjustment in Tycho first, I'll try to take a look. As always if one want to 
provide an [integration-test](https://github.com/eclipse/tycho/wiki#providing-an-integration-test) to demonstrate the issue that would be helpful.

Reply:
I have already created an according integration test (on https://github.com/HeikoKlare/tycho/tree/junit5-with-linked-files) and will provide a describing issue and a PR with that test to the Tycho project tomorrow.

Reply:
Tycho issue is now documented along with a reproducer PR: https://github.com/eclipse-tycho/tycho/issues/3803